components/Repositories/AddToWorkspaceModal.tsx

```diff
@@ -32,6 +33,10 @@ export default function AddToWorkspaceModal({ repository, isOpen, onCloseModal }
   }, []);
 
   const addRepositoryToWorkspace = async () => {
+    posthog.capture(`Repo Pages: added to ${workspaceId === "new" ? "a new" : "existing"} workspace`, {
```

Comment:
refactor (blocking): prefer the `usePosthog` hook instead as we already have a [Posthog provider configured](https://github.com/open-sauced/app/blob/95774d898160e19a2d427b5dc4cd8d9474f625a0/pages/_app.tsx#L152-L162). See https://posthog.com/tutorials/single-page-app-pageviews and an example usage at https://github.com/open-sauced/app/blob/95774d898160e19a2d427b5dc4cd8d9474f625a0/components/ListHeader/list-header.tsx